nautobot/ipam/views.py

```diff
@@ -399,17 +399,15 @@ class PrefixListView(generic.ObjectListView):
     filterset_form = forms.PrefixFilterForm
     table = tables.PrefixDetailTable
     template_name = "ipam/prefix_list.html"
-    # `queryset` is implemented as a property, see below
 
     def __init__(self, *args, **kwargs):
         # Set the internal queryset value
         self._queryset = None
+        self.queryset = self.set_queryset()
         super().__init__(*args, **kwargs)
 
-    @property
-    def queryset(self):
+    def set_queryset(self):
```

Comment:
I wonder if the actual issue here is at line 418/420 below with `if self._queryset:` - I think an empty queryset evaluates as boolean False? It might be that a simpler fix for this issue might be simply to change that line to `if self._queryset is not None:` instead.

Reply:
Thanks, I'll test this solution.

Reply:
Tested adding "is not None" and updated to add back in property/setter. The suggested change also fixed the issue, but in a much simpler way.

Reply:
Yep, this is the right way to do this because we will eventually move to the `get_queryset()` pattern and property will go away. This is really an oversight on my part in implementing this, so the test case is really helpful.